meson_options.txt

```diff
@@ -4,3 +4,8 @@ option(
   type: 'string',
   value: '/usr/share/profile.d'
 )
+option(
+  'tmpfilesdir',
```

Comment:
It will be good to call it `tmpfiles_dir` for the sake of consistency with the other option (ie., `profile_dir`) in `meson_options.txt`.

In fact, `tmpfiles_dir` is a thing in `systemd.pc`. I wonder why.